[2023-08-28T14:44:51.606+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-28T14:44:51.615+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-08-25T00:00:00+00:00 [queued]>
[2023-08-28T14:44:51.615+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-08-28T14:44:51.616+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-08-28T14:44:51.616+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-08-28T14:44:51.630+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-08-25 00:00:00+00:00
[2023-08-28T14:44:51.635+0000] {standard_task_runner.py:55} INFO - Started process 91 to run task
[2023-08-28T14:44:51.639+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2023-08-25T00:00:00+00:00', '--job-id', '61', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp_0oqrnfq']
[2023-08-28T14:44:51.641+0000] {standard_task_runner.py:83} INFO - Job 61: Subtask create_table
[2023-08-28T14:44:51.700+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2023-08-25T00:00:00+00:00 [running]> on host 22e2fb7a3f74
[2023-08-28T14:44:51.764+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-08-25T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-08-25T00:00:00+00:00
[2023-08-28T14:44:51.773+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-08-28T14:44:51.776+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
            firstname TEXT NOT NULL,
            lastname TEXT NOT NULL,
            country TEXT NOT NULL,
            username TEXT NOT NULL,
            password TEXT NOT NULL,
            email TEXT NOT NULL
            ) ;
         , parameters: None
[2023-08-28T14:44:51.778+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-08-28T14:44:51.785+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230825T000000, start_date=20230828T144451, end_date=20230828T144451
[2023-08-28T14:44:51.810+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-08-28T14:44:51.826+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
